src/Application/Responses/FileResponse.php

```diff
@@ -40,6 +40,9 @@ public function __construct(string $file, string $name = null, string $contentTy
 		if (!is_file($file)) {
 			throw new Nette\Application\BadRequestException("File '$file' doesn't exist.");
 		}
+		if (!is_readable($file)) {
+			throw new Nette\Application\BadRequestException("File '$file' doesn't readable.");
```

Comment:
is not readable sounds better

Reply:
I'm not sure how useful the `is_readable` check is when you check `fopen` result later anyway.

Reply:
@JanTvrdik The sooner the better :-) But I don't insist on it.